TestFoundation/TestNSBundle.swift

```diff
@@ -115,12 +115,12 @@ class TestNSBundle : XCTestCase {
             
             // Put some resources in the bundle
             for n in _bundleResourceNames {
-                NSFileManager.defaultManager().createFile(atPath: bundlePath + "/" + n, contents: nil, attributes: nil)
+                _ = NSFileManager.defaultManager().createFile(atPath: bundlePath + "/" + n, contents: nil, attributes: nil)
```

Comment:
Im happy to make such a change.  I imagine that a) that would need to go through swift evolution b) should be a follow up change.